main/src/ca/uwaterloo/flix/language/phase/unification/BddFormula.scala

```diff
@@ -15,10 +15,105 @@
  */
 package ca.uwaterloo.flix.language.phase.unification
 
+import ca.uwaterloo.flix.language.ast.{SourceLocation, Symbol, Type}
+import ca.uwaterloo.flix.util.InternalCompilerException
+import ca.uwaterloo.flix.util.collection.Bimap
+
+import scala.collection.immutable.SortedSet
+import org.sosy_lab.pjbdd.api.{Builders, Creator, DD}
+
 object BddFormula {
```

Comment:
Overall I would like some longer comments. 

Essentially I just a bit more information as a BDD novice to understand what's going on.